### PR TITLE
doc: update helm documentation

### LINKED ISSUE #1073: HELM install doc does not mention which repo to add
# Describe the feature you'd like to have #
Improve install doc

What new functionality do you want?
# What is the value to the end user? (why is it a priority?) #
Improve user experience

The [HELM install doc](https://github.com/ceph/ceph-csi/blob/master/charts/ceph-csi-cephfs/README.md) does not mention which repo to add

How would the end user gain value from having this feature?

# How will we know we have a good solution? (acceptance criteria) #

helm repo add ceph-csi https://ceph.github.io/csi-charts

Add a list of criteria that should be met for this feature to be useful

# Additional context #

### PR DESCRIPTION
**Pull request by Madhu-1**
# Describe what this PR does #

Updated Helm documentation on how to install charts from release repo

Fixes #1073 

Signed-off-by: Madhu Rajanna <madhupr007@gmail.com>

### CODE REVIEW

File: charts/ceph-csi-cephfs/README.md [hunk 1]
```diff
@@ -2,7 +2,25 @@
...
+
+```console
+helm repo add ceph-csi https://ceph.github.io/csi-charts
+```
+
+## Install from local Chart
+
+we need to enter into the directory where all charts are present
+
+```console
```
Comment (nixpanic) [inline charts/ceph-csi-cephfs/README.md:17]: This assumes the ceph-csi git repository is already cloned and the current working directory. I am not sure that is reasonable. Maybe users need the commands to clone the repository too?

Comment (Madhu-1) [reply charts/ceph-csi-cephfs/README.md:17]: This assumes that the user has cloned the repo. do you want me to add code to clone and check out the repo? I am assuming that user will  know these steps

Comment (nixpanic) [reply charts/ceph-csi-cephfs/README.md:17]: Ok, if that is your assumption, no need to add it now. If someone wants to add them in the future, that can always be done.

**Comment by humblec** [comment]
@Madhu-1 travis fails again

**Comment by Madhu-1** [comment]
> @Madhu-1 travis fails again

due to other issues, restarted the CI

**Pull request merged**